raven/processors.py

```diff
@@ -66,31 +66,23 @@ def filter_stacktrace(self, data, **kwargs):
             frame.pop('vars', None)
 
 
-class SanitizePasswordsProcessor(Processor):
+class SanitizeKeysProcessor(Processor):
     """
-    Asterisk out things that look like passwords, credit card numbers,
-    and API keys in frames, http, and basic extra data.
+    Asterisk out things that correspond to a configurable set of keys.
     """
     MASK = '*' * 8
-    FIELDS = frozenset([
-        'password',
-        'secret',
-        'passwd',
-        'authorization',
-        'api_key',
-        'apikey',
-        'sentry_dsn',
-        'access_token',
-    ])
-    VALUES_RE = re.compile(r'^(?:\d[ -]*?){13,16}$')
+
+    def __init__(self, client):
+        super(SanitizeKeysProcessor, self).__init__(client)
+        fields = getattr(client, 'sanitize_keys')
```

Comment:
Could you move this into a `get_fields` method, or rather `get_keys` to be more consistent with naming.